src/kartograf/mapping_metrics/metric_mapping_rmsd.py

```diff
@@ -82,4 +97,14 @@ def get_score(self, mapping, accepted_distance_rmsd:float = 0, k_hook=10**0, T=2
         float
             normalized score between 0 and 1.
         """
-        return float(np.round(self.get_rmsd_p(mapping, accepted_distance_rmsd = accepted_distance_rmsd, k_hook= k_hook, T= T),2))
+        return float(
+            np.round(
+                self.get_rmsd_p(
+                    mapping,
+                    accepted_distance_rmsd=accepted_distance_rmsd,
+                    k_hook=k_hook,
+                    T=T,
+                ),
+                2,
```

Comment:
I'm not sure this is more readable than before, I have no idea what this `2` is doing floating there.  Rather than split one very complex line over many lines, just break the different function calls onto different lines.  It will take the same amount of space but you can logically break down what this line is doing